tests/unit/test_charm.py

```diff
@@ -369,19 +370,19 @@ def test_get_pebble_config(self):
             },
         }
 
-        r = self.harness.charm._get_pebble_config(mock_event)
+        r = self.harness.charm._get_gunicorn_pebble_config(mock_event)
         self.assertEqual(r, expected_ret)
 
-    def test_get_pebble_config_error(self):
-        """Test the _get_pebble_config function when throwing an error."""
+    def test_get_gunicorn_pebble_config_error(self):
+        """Test the _get_gunicorn_pebble_config function when throwing an error."""
```

Comment:
Instead of comparing the whole error message, you can find in the result just a substing or some keywords to have more maintainable tests.